src/components/expense-list-section/expense-list/expense-item/expense-item.tsx

```diff
@@ -0,0 +1,53 @@
+import { EditOutlined, UserOutlined } from "@ant-design/icons";
+import "./expense-item.css";
+import { Expense } from "../../../../@types/expense";
+
+export const ExpenseItem = ({
+  item,
+  className,
```

Comment:
needed?

Reply:
In other branch I will fix it
https://github.com/ofir-zeitoun/expenses-client/issues/33